libvast/vast/detail/stable_map.hpp

```diff
@@ -34,10 +36,4 @@ struct stable_map_policy {
   }
 };
 
-/// A map abstraction over an unsorted `std::vector`.
-template <class Key, class T,
-          class Allocator = std::allocator<std::pair<Key, T>>>
-using stable_map = vector_map<Key, T, Allocator, stable_map_policy>;
```

Comment:
Here should be a comment that directs the reader to `vast/fwd.hpp`.

Reply:
Done.